skills/gh-address-comments/scripts/install_completions.py

```
@@ -0,0 +1,258 @@
+        source_hint = f"source {dest_file}  # add to ~/.bashrc if not auto-sourced"
+
+    if args.print_only:
+        print(script)
+        return
+
+    os.makedirs(dest_dir, exist_ok=True)
+    with open(dest_file, "w") as f:
+        f.write(script)
+
+    print(f"✓ Installed {shell} completions to {dest_file}")
+    print(f"\nIf completions don't work immediately, add this to your shell config:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)